src/Components/List.js

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+
+
+class List extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state ={
+            viewList: (item) => {
+                return item
+            }
+        }
+    }
+
+    render(){
+    return(
+        <section className="main">
+            <input id="toggle-all" className="toggle-all" type="checkbox" />
+            <label htmlFor="toggle-all">Mark all as complete</label>
+            <ul className="todo-list">
+                {
+
+                    this.props.todoList.filter(this.state.viewList).map((item) => {
```

Comment:
Create `filterItems` method which takes `this.props.todoList` and applies filter on it.
```suggestion
                    this.filterItems().map((item) => {
```